### DISCUSSION TITLE
Data Usage policy

### DISCUSSION THREAD

**Discussion by Deepankar-98** [tecoholic/ner-annotator#114] [Q&A]
Does this tools store, save the data that is being uploaded and annotated using either Web or Desktop version?

**Comment by alvi-khan** [comment]
No it doesn't. Neither version actually uploads any files at all. The files are always opened on the client side.

**Reply by Deepankar-98** [reply]
Thanks for the confirmation.
Is there any documentation or some other source where it is mentioned?
I am just trying to get all approvals for using this tool

**Reply by alvi-khan** [reply]
No unfortunately. We don't have any official documentation regarding the data usage policy at the moment. I can see how it could be useful in academic and professional contexts though. I'll look into adding this soon.

**Reply by alvi-khan** [reply]
@Deepankar-98 We've added one: https://github.com/tecoholic/ner-annotator/blob/main/PRIVACY-POLICY.md
